runtime/doc/intro.txt

```diff
@@ -373,8 +373,10 @@ notation	meaning		    equivalent	decimal value(s)	~
 <kMinus>	keypad -			*keypad-minus*
 <kMultiply>	keypad *			*keypad-multiply*
 <kDivide>	keypad /			*keypad-divide*
-<kEnter>	keypad Enter			*keypad-enter*
 <kPoint>	keypad Decimal point		*keypad-point*
+<kComma>	keypad comma key		*keypad-comma*
+<kEqual>	keypad equal key		*keypad-equal*
```

Comment:
following the pattern from above, these would be:

```
<kComma>	keypad ,		*keypad-comma*
<kEqual>	keypad =		*keypad-equal*
```

Reply:
:+1: